packages/web/src/components/ProjectCard/ProjectCard.tsx

```diff
@@ -0,0 +1,62 @@
+import { Heading, Text, Flex, Box, Link } from '@chakra-ui/react';
+import { FiExternalLink } from 'react-icons/fi';
+import { motion } from 'framer-motion';
+import { Project } from '@hangar/shared';
+
+type ProjectCardProps = {
+  project: Project;
+};
+
+export const ProjectCard: React.FC<ProjectCardProps> = ({ project }) => {
+  const { name, description, location, repoUrl } = project;
+
+  return (
+    <motion.div
```

Comment:
I personally don't see the value in adding a hover effect to a static card, but that's a personal opinion. If you think this looks good, we can leave it in...

Reply:
it looks, but we can always remove it later